src/command.cpp

```diff
@@ -89,7 +89,7 @@ int Command::runCommandLineApp(std::function<int()>&& a_callback) {
   FeatureList::instance()->initialize();
 
   qInstallMessageHandler(LogHandler::messageQTHandler);
-  logger.info() << "MozillaVPN" << APP_VERSION;
+  logger.info() << "Core" << APP_VERSION;
```

Comment:
I think this should stay as "MozillaVPN" as a name for our product, or at least until we come up with a name for this expanded product concept that's more than just a VPN.